doc/manual/performance-tips.rst

```diff
@@ -932,6 +927,13 @@ On a computer with a 2.4GHz Intel Core i5 processor, this produces::
     GFlop        = 1.9467069505224963
     GFlop (SIMD) = 17.578554163920018
 
+Certain preconditions need to be met before using some of these macros.
+Both :obj:`@simd` and :obj:`@inbounds` do a simple rewrite of the expression 
+and delegate the optimizations to compiler, much like ``:meta`` expressions.
+This means they merely give the compiler license to optimize. Whether
+it actually does so depends on the compiler. As consequence how you 
```

Comment:
"As consequence" is not grammatical.  Actually, I think you can just delete the "As consequence..." sentence.